pyvista/plotting/helpers.py

```diff
@@ -1,19 +1,38 @@
 """This module contains some convenience helper functions."""
+from typing import Optional, List, Union
 
+import itkwidgets
```

Comment:
We can't import this here as `itkwidgets` is an optional feature.

Reply:
I am aware, but it must be imported for type hints. Don't really have a good solution yet, could just leave this one out.

Reply:
`itkwidgets` doesn't have typing hints anyway, so let's not worry about this.